database/postgresDB.js

```diff
@@ -0,0 +1,14 @@
+const { Sequelize } = require('sequelize');
+
+const db = new Sequelize({
+  database: 'summary',
+  username: 'root',
+  password: 'root',
```

Comment:
I would make sure you move this username and password to a config file or env vars.. (I need to do the same)